pkg/rebuild/rebuild/storage.go

```
@@ -87,18 +89,24 @@ type LocatableAssetStore interface {
 	URL(a Asset) *url.URL
 }
 
+func logWrap(f func() error) {
+	if err := f(); err != nil {
+		log.Print(err)
+	}
+}
+
 // AssetCopy copies an asset from one store to another.
 func AssetCopy(ctx context.Context, to, from AssetStore, a Asset) error {
 	r, err := from.Reader(ctx, a)
 	if err != nil {
 		return errors.Wrap(err, "from.Reader failed")
 	}
-	defer r.Close()
+	defer logWrap(r.Close)
 	w, err := to.Writer(ctx, a)
 	if err != nil {
 		return errors.Wrap(err, "to.Writer failed")
 	}
-	defer w.Close()
+	defer logWrap(w.Close)
```

you could also actually return the error here, right?